lib/base/RestException.js

```diff
@@ -3,14 +3,34 @@
 class RestException extends Error {
```

Comment:
We currently are working on a release candidate for v4 on the 4.0.0-rc branch. Do you mind submitting the changes to that branch (which no longer has any js files, all ts) and add unit tests to validate functionality?

Reply:
okay. i'll close this pull request then